APIURLS.md

```diff
@@ -0,0 +1,110 @@
+Admin Features:
+    Manage Project:
+        Projects Database:
+            localhost:8080/API/v1/project/id/1
+            [Post, Put, Get, Delete]
+        Create Project:
+            POST localhost:8080/API/v1/projects/id
+        Search Project:
+            GET localhost:8080/API/v1/projects?search=id
+        Delete Project:
+            Delete localhost:8080/API/v1/projects/id
+        List Project Table:
+            GET localhost:8080/API/v1/projects
+    User Management:
+        Users Database:
+            localhost:8080/API/v1/User/id/1
+            [Post, Put, Get, Delete]
+            localhost:8080/API/v1/Users
+        Invite Member: (send user invitation by email)
+            POST localhost:8080/API/v1/projects/id/Users/id
+        List User Table:
+            GET localhost:8080/API/v1/projects/id/Users
+        User status:
+            GET localhost:8080/API/v1/Users/id/status
+        Update User Last active:
+            GET localhost:8080/API/v1/Users/id/lastActive
+            PATCH localhost:8080/API/v1/Users/id

```

Comment:
PATCH localhost:8080/API/v1/Users/id/lastActive

Reply:
What is the request of the PATCH (in json format)?
@Lambton42 @ChunhuiY @zoeee06

Reply:
The request of this specific PATCH URL should be:
PATCH /API/v1/Users/12345/lastActive HTTP/1.1
Host: localhost:8080
Content-Type: application/json

{
  "lastActive": "2024-04-04 12:34:56"
}